src-tauri/src/mod_management/legacy.rs

```diff
@@ -100,6 +100,17 @@ pub fn parse_installed_mods(
         // Get directory path
         let mod_directory = directory.to_str().unwrap().to_string();
 
+        // This is a stupid way to show a legacy installed mod as outdated by simply giving back a wrong version number
+        if thunderstore_mod_string.is_some() {
+            // Parse the string
+            let mut parsed_string: ParsedThunderstoreModString =
+                thunderstore_mod_string.clone().unwrap().parse().unwrap();
+            // Set version number to `0.0.0`
+            parsed_string.version = "0.0.0".to_string();
+            // And store new string back in original variable
+            thunderstore_mod_string = Some(parsed_string.to_string())
+        }
```

Comment:
perhaps just move this into the match case above ;)

Reply:
Wouldn't I have to duplicate the code then? Cause the `match` matches for like two cases
1. checking if it's in `mods.json`
2. checking if it there's a `manifest.json` and a `thunderstore_author.txt`